homeassistant/components/bitcoin/sensor.py

```diff
@@ -148,7 +148,7 @@ def update(self):
         elif self.type == "total_btc":
             self._state = "{0:.2f}".format(stats.total_btc * 0.00000001)
```

Comment:
What about this (and other spots that still round to two decimal places)?

Reply:
That is correct. The amount of Bitcoin a miner receives varies every halving. The reward is now 12.5 BTC for every block. From approximately may it wil be 6.25 BTC per block.